pkg/controllermanager/controller/controllerregistration/seed_control.go

```diff
@@ -36,6 +36,7 @@ func (c *Controller) seedAdd(obj interface{}) {
 		return
 	}
 	c.seedQueue.Add(key)
+	c.controllerRegistrationSeedQueue.Add(key)
```

Comment:
Should we only do that if the (nginx) ingress spec in the seed has changed?

Reply:
I addressed this in https://github.com/gardener/gardener/pull/3131/commits/40b189076be9b0349d89b0a5b6f4fbae73e1ff0b

Unsure if we want to keep it though.